server.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+const app = express()
+const port = 3000
```

Comment:
Consider using environment variables for the port number. This allows flexibility in different environments. You can use packages like 'dotenv' for managing environment variables. [important]